tests/config/issue-4041.toml

```diff
@@ -0,0 +1 @@
+wrap_comments=true
```

Comment:
I don't think this file is necessary.

I would use a comment like `//rustfmt-wrap_comments: true`. The reason why `wrap_comments=true` isn't having any affect in the target file is because no line exceeds the default max width of  100 characters. I think we should add a `tests/source/issue-4041.rs` that is unformatted and has `+` lines that exceed 100 chars long.